addons/account/populate/res_company.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from odoo import models
+import random
+from functools import lru_cache
+
+
+_logger = logging.getLogger(__name__)
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    def _populate(self, size):
+        @lru_cache()
+        def search_coa_ids(currency_id):
+            return self.env['account.chart.template'].search([('currency_id', '=', currency_id)])
+
+        records = super()._populate(size)
+        _logger.info('Loading Chart Template')
+        default_chart_templates = self.env['account.chart.template'].search([], limit=1)
+        if not default_chart_templates:
+            # TODO install l10n_generic_coa ?
+            pass
+        for company in records[:3]:
```

Comment:
Because I want to test with a lot of account.move.line, and doing it on all the companies is going to be really, really, really long to create. It already takes a lot of time as it is... (I understand we will be able to download dumps, but it REALLY will be long to create)
When we want to test performances, the UI, etc., we only need to do it with a few companies at the same time.

Reply:
Maybe you could create a variable number of move lines for each companies, and a lot for main_company?